custom/enikshay/nikshay_datamigration/models.py

```diff
@@ -286,8 +286,8 @@ class Outcome(models.Model):
     @property
     def hiv_status(self):
         return {
-            None: None,
-            'NULL': None,
+            None: 'unknown',
```

Comment:
@proteusvacuum is this needed for 1.0?  Wondering if I can run the migration today and add the drtb-hiv-referral cases later or if that will break the 1.0 app.